app/services/validation_beta_service.rb

```diff
@@ -1,6 +1,31 @@
 # frozen_string_literal: true
 
 class ValidationBetaService
+  def remind_sits_to_add_participants
+    empty_school_cohorts = SchoolCohort
+      .where(induction_programme_choice: %i[full_induction_programme core_induction_programme not_yet_known])
+      .where(opt_out_of_updates: false)
+      .where.not(id: ParticipantProfile::ECF.select(:school_cohort_id))
+
+    emails_sent = []
+
+    School.where(id: empty_school_cohorts.select(:school_id)).includes(:induction_coordinators).find_each do |school|
+      school.induction_coordinators.each do |sit|
+        next if emails_sent.include?(sit.email)
```

Comment:
```suggestion
        next if emails_sent.include?(sit.email)
```

This isn't idempotent because we aren't persisting anything against the SIT. If the instance dies while you're running this, can you run it again?